native/iparmq.go

```diff
@@ -69,8 +69,11 @@ func (Implementation) Iparmq(ispec int, name, opts string, n, ilo, ihi, lwork in
 
 	case 14:
 		// Skip a multi-shift QR iteration whenever aggressive early
-		// deflation finds at least (nibble*(window size)/100)
-		// deflations.
+		// deflation finds at least nibble*(window size)/100 deflations.
+		// The default, small value reflects the expectation that the
```

Comment:
Either delete the comma or add a closing parenthetic comma after "small". The sentence parse is a lttle difficult, but I don't see a way to improve it.